velox/dwio/common/tests/utils/DataSetBuilder.cpp

```diff
@@ -149,6 +149,30 @@ DataSetBuilder& DataSetBuilder::withNullsForField(
   return *this;
 }
 
+DataSetBuilder& DataSetBuilder::withTimestampDistributionForField(
```

Comment:
Do you have particular reason why the default generated data is not good enough?

Reply:
Supported Timestamp type in `withIntDistributionForField` and changed to use it instead.